easybuild/easyconfigs/c/CP2K/CP2K-5.1-intel-2017b.eb

```diff
@@ -0,0 +1,42 @@
+name = 'CP2K'
+version = '5.1'
+
+homepage = 'http://www.cp2k.org/'
+description = """CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular
+ simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different
+ methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and
+ classical pair and many-body potentials. """
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'pic': True, 'precise': True}
```

Comment:
The `'precise': True` was helping significantly with getting more passed tests in the test suite, especially on Intel Haswell, but I'm not sure what kind of impact this has on performance.

@tovrstra Are you or someone else in your group up for taking a look at this?

Reply:
Note: do use the updated easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/1293...

Reply:
Should we also look at the tests that come with CP2K or rather tests different types of simulations that we are currently doing also with older versions of CP2K?

Reply:
Well, both, but specifically w.r.t. the use of `precise` I'm mainly concerned with performance of 'real' runs.

Reply:
OK. I'll already forward it to the involved colleagues. I may also try some of this myself today.

Reply:
Hmm, I checked this again today, and the impact of enabling `precise` is not as I reported earlier. Here's what I'm seeing now, on an Intel Haswell system (and the updated CP2K easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/1293):

* with `precise` enabled: 1 failed test, 13 wrong tests; total time for testing: 3559s
*with `precise` disabled (the default setting): 1 failed test, 1 wrong test; total time for testing: 2897s

So, the impact of enabling `precise` on performance seems quite substantial (about 20%), while it doesn't help at all with the test results (it even makes it a bit worse)...

Hence, I'll update this PR to drop the `'precise': True`...